lib/createDIContainer.js

```diff
@@ -88,10 +87,18 @@ const queryHandlerFactory = require('./abci/handlers/queryHandlerFactory');
  * @param {string} options.CORE_JSON_RPC_PORT
  * @param {string} options.CORE_JSON_RPC_USERNAME
  * @param {string} options.CORE_JSON_RPC_PASSWORD
+ * @param {string} options.IDENTITY_ENABLE_LOCK_TX_ONE_BLOCK_CONFIRMATION_FALLBACK
  *
  * @return {AwilixContainer}
  */
 async function createDIContainer(options) {
+  const additionalDppOptions = {
+    identities: {
+      enableLockTxOneBlockConfirmationFallback: !!options
```

Comment:
Are options mandatory? Because if not, that line will cause a crash

Reply:
Yeah, they are.